### ISSUE TITLE
Document pip / easy_install workarounds

### ISSUE THREAD

**Issue by slinkp** [openplans/openblock#156]
Some installation problems are just hard to prevent; document the workarounds instead:
https://groups.google.com/group/ebcode/browse_thread/thread/593c8068a8e89778
... essentially, `easy_install distribute>=0.6.16` before you do anything else fixes some bogus installation failures.

**Comment by slinkp** [comment]
Also, when depending on eg. a specific git commit in requirements.txt, we've found painfully that this works:

```
pip install -r ebpub/requirements.txt
pip install -e ebpub
```

but this does NOT work:

```
pip install -r ebpub/requirements.txt -e ebpub

```

The latter approach causes the right package to get downloaded, but the "-e ebpub" part apparently causes us to spin off into setuptools land where it doesn't know anything about the git download so it goes and searches pypi for whatever's specified in setup.py, and _that_ is what gets installed.
So people keep getting wrong versions of things.

The one-line version that doesn't work is listed in our docs several times. Ugh.

(Filed as a pip bug: https://github.com/pypa/pip/issues/318)

**Comment by slinkp** [comment]
(In [e7d7820aec690ae6533a2e84e8e41da3c90d976d]) Docs re. install problems, closes #162

**Comment by slinkp** [comment]
Ticket imported from Trac:
 http://developer.openblockproject.org/ticket/162
Reported by: slinkp
